happi/tests/test_backends.py

```diff
@@ -147,12 +147,13 @@ def test_json_initialize():
 
 @requires_questionnaire
 def test_qs_find(mockqsbackend):
-    assert len(mockqsbackend.find(beamline='TST', multiples=True)) == 6
+    assert len(mockqsbackend.find(beamline='TST', multiples=True)) == 14
     assert len(mockqsbackend.find(name='sam_r', multiples=True)) == 1
 
 
 @requires_questionnaire
 def test_qsbackend_with_client(mockqsbackend):
     c = Client(database=mockqsbackend)
-    assert len(c.all_devices) == 6
-    assert all([isinstance(d, Motor) for d in c.all_devices])
+    assert len(c.all_devices) == 14
+    assert all([isinstance(d, Motor) or isinstance(d, Trigger)
```

Comment:
You should check you got the proper number of each i.e
```python
device_types = [device.__class__ for device in c.all_devices]
assert device_types.count(Trigger) == 4
...
```

Reply:
Good call, added this in 08d8c41